src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+class DukeGotNoArgumentsException extends IllegalArgumentException {
+    public DukeGotNoArgumentsException(String message) {
+        super("Please reenter your tasks in the format " + message + "\nFill in all blanks.");
```

Comment:
typo: should be re-enter

Reply:
Thank you for all your comments, they are definitely useful, will take note! :)